### PR TITLE
add new event template

### PR DESCRIPTION
**Pull request by areisle**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/venue-host.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 name: Host request
 about: Host an upcoming VanJS event at your venue
 title: 'Host request: '
-labels: Host
```
Comment (areisle) [inline .github/ISSUE_TEMPLATE/venue-host.md]: I'm not sure why it's showing changes to these files... it must have automatically updated them?

**Comment by areisle** [comment]
> We should probably also add:
> 
> * MC host
> * Event slides

added in [33f331f](https://github.com/VanJS/meetup/pull/42/commits/33f331f2f2bcb3c1b2cd5c04c13f14435a967beb)

File: .github/ISSUE_TEMPLATE/venue-host.md (hunk 1 above)
Comment (jagonalez) [reply .github/ISSUE_TEMPLATE/venue-host.md]: looks like it removed it since the label name was in correct. Should be correct now

**Comment by jagonalez** [comment]
Just added a few changes, here's the rendered version: https://github.com/VanJS/meetup/blob/add-event-template/.github/ISSUE_TEMPLATE/event.md

**Pull request merged**